gapic/src/main/com/google/gapid/views/CommandTree.java

```diff
@@ -341,7 +341,8 @@ protected void load(CommandStream.Node node, Runnable callback) {
                 string, string.identifierStyle());
           }
         } else {
-          string.append(Formatter.firstIndex(data.getCommands()) + ": ", string.defaultStyle());
+          String index = shouldShowLastIndex(data) ? Formatter.lastIndex(data.getCommands()) : Formatter.firstIndex(data.getCommands());
```

Comment:
Can we use the ID of the `representation` instead here so we don't have to teach the frontend that draws are different?

Reply:
see CommandStream.java:375 or so